deepfence_frontend/apps/dashboard/src/features/registries/layouts/RegistryConnectorLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import { HiArrowSmLeft } from 'react-icons/hi';
```

Comment:
maybe delete?

Reply:
layout is commonly used for all connector forms